files/zh-cn/web/api/rtcpeerconnection/index.md

```diff
@@ -3,250 +3,149 @@ title: RTCPeerConnection
-- {{domxref("RTCPeerConnection.setRemoteDescription()")}}
-  - : 改变与连接相关的远端描述。这个描述定义了连接的属性，例如：连接的编码方式。连接会受到它的改变的影响，而且连接必须能同时支持新的以及旧的描述。这个方法可以接收三个参数，一个{{domxref("RTCSessionDescription")}} 对象包含设置信息，还有两个回调函数，它们分别是方法调用成功以及失败的回调函数。
-- {{domxref("RTCPeerConnection.updateIce()")}}
-  - : 更新 ICE 服务器时调用的方法。
-- {{domxref("RTCPeerConnection.addIceCandidate()")}}
-  - : 添加 iceCandidate 时调用的方法。
-- {{domxref("RTCPeerConnection.getConfiguration()")}}
-  - : 获取配置信息时调用的方法。
-- {{domxref("RTCPeerConnection.getLocalStreams()")}}
-  - : 返回连接的本地媒体流数组。这个数组可能是空数组。
-- {{domxref("RTCPeerConnection.getRemoteStreams()")}}
-  - : 返回连接的远端媒体流数组。这个数组可能是空数组。
-- {{domxref("RTCPeerConnection.getStreamById()")}}
-  - : 返回连接中与所给 id 匹配的媒体流 {{domxref("MediaStream")}}，如果没有匹配项，返回 null。
-- {{domxref("RTCPeerConnection.addStream()")}}
-  - : 添加一个媒体流 {{domxref("MediaStream")}}作为本地音频或视频源。如果本地端与远端协调已经发生了，那么需要一个新的媒体流，这样远端才可以使用它。
-- {{domxref("RTCPeerConnection.removeStream()")}}
-  - : 将一个作为本地音频或视频源的媒体流 {{domxref("MediaStream")}}移除。如果本地端与远端协调已经发生了，那么需要一个新的媒体流，这样远端才可以停止使用它。
-- {{domxref("RTCPeerConnection.close()")}}
-  - : 关闭一个 RTCPeerConnection 实例所调用的方法。
-- {{domxref("RTCPeerConnection.createDataChannel()")}}
-  - : 在一条连接上建立一个新的{{domxref("RTCDataChannel")}}（用于数据发送）。这个方法把一个数据对象作为参数，数据对象中包含必要的配置信息。
-- {{domxref("RTCPeerConnection.createDTMFSender()")}}
-  - : 创建一个新的与特殊的{{domxref("MediaStreamTrack")}}相关的{{domxref("RTCDTMFSender")}}，可以在连接上发送{{Glossary("DTMF")}}手机信号。
-- {{domxref("RTCPeerConnection.getStats()")}}
-  - : 生成一个新的{{domxref("RTCStatsReport")}}，它包含连接相关的统计信息。
-- {{domxref("RTCPeerConnection.setIdentityProvider()")}}
-  - : 根据所给的三个参数设置身份提供者（IdP)，这三个参数是它的名称，通信所使用的协议（可选），以及一个可选的用户名。只有当一个断言被需要时，这个 IdP 才会被使用。
-- {{domxref("RTCPeerConnection.getIdentityAssertion()")}}
-  - : 初始化身份断言的收集，只有当{{domxref("RTCPeerConnection.signalingState", "signalingState")}}的值不为"closed"时，它才有效。它自动完成，在需求发生前调用它是最好的选择。
+_继承 {{DOMxRef("EventTarget")}} 的方法。_
+
+- {{DOMxRef("RTCPeerConnection.addIceCandidate", "addIceCandidate()")}}
+  - : 向 `RTCPeerConnection` 的远程描述中添加一个新的远程候选者，描述连接的远程端的状态。
+- {{DOMxRef("RTCPeerConnection.addTrack", "addTrack()")}}
+  - : 向要传输给对方的轨道集合中添加一个新的 {{DOMxRef("MediaStreamTrack")}} 。
+- {{DOMxRef("RTCPeerConnection.addTransceiver", "addTransceiver()")}}
+  - : 创建一个新的 {{DOMxRef("RTCRtpTransceiver")}} 并将其添加到与连接关联的收发器集合中。
+    每个收发器代表一个双向流，分别与之关联一个 {{DOMxRef("RTCRtpSender")}} 和 {{DOMxRef("RTCRtpReceiver")}} 。
+- {{DOMxRef("RTCPeerConnection.close", "close()")}}
+  - : 关闭当前连接，释放所有资源。
+- {{DOMxRef("RTCPeerConnection.createAnswer", "createAnswer()")}}
+  - : 发起创建 {{Glossary("SDP")}} 应答（answer），以响应来自远程对等方的邀请（offer），在 `WebRTC` 连接的邀请（offer）/应答（answer）协商过程中。应答包含会话中已附加的媒体、编解码器和浏览器支持的其它选项，以及已收集的 {{Glossary("ICE")}} 候选者的信息。
+- {{DOMxRef("RTCPeerConnection.createDataChannel", "createDataChannel()")}}
+  - : 创建一个与远程对等连接的新通道，该通道可以传输任何类型的数据。
+    例如图像、文件传输、文本聊天、游戏更新包等。
```

Comment:
考虑下 This can be useful for back-channel content 如何翻译 https://github.com/mdn/content/blob/c0e47421330d58d462e8240405a5738332514d1c/files/en-us/web/api/rtcpeerconnection/index.md?plain=1#L88C5-L88C48

```suggestion
  - : 创建一个与远程对等连接的新通道，该通道可以传输任何类型的数据。例如图像、文件传输、文本聊天、游戏更新包等。
```